wiki/Glossary/Combo_(score_multiplier)/en.md

```diff
@@ -1,9 +1,10 @@
 ---
+stub: true
 tags:
   - max combo
 ---
 
-# Combo (Score multiplier)
+# Combo (score multiplier)
```

Comment:
are we sure we want to use "score multiplier" in parantheses instead of "multiplier effect"?